framework/Core/CLI/Uninstall.php

```diff
@@ -24,11 +24,13 @@ public function handle(Kernel $kernel, array $argv) {
 		if (file_exists(__ROOT__ . '/production.lock'))
 			throw new \RuntimeException('Can not run uninstall script because production.lock exists.' . PHP_EOL);
 
-		printf('THIS METHOD WILL EMPTY DATABASE \'%s\', ARE YOU ABSOLUTELY SURE? [y/n]' . PHP_EOL, $kernel->getConfig()->db_database);
-		$line = trim(fgets(STDIN));
+		if (count($argv) < 1 || $argv[0] !== 'y') {
```

Comment:
```suggestion
		if (count($argv) < 1 || $argv[1] !== 'y') {
```
Shouldn't this be 1? Argument zero is usually the application itself, e.g. `/var/git/SmoothPHP/smoothphp`.

Reply:
I have tested, and when running "smoothphp reinstall y" argument zero = y